src/OpenTK.Windowing.GraphicsLibraryFramework/Enums/Keys.cs

```diff
@@ -259,16 +259,6 @@ public enum Keys
         /// </summary>
         GraveAccent = 96 /* ` */,
 
-        /// <summary>
-        /// Non US keyboard layout key 1.
-        /// </summary>
-        World1 = 161 /* non-US #1 */,
-
-        /// <summary>
-        /// Non US keyboard layout key 2.
-        /// </summary>
-        World2 = 162 /* non-US #2 */,
-
```

Comment:
Why did these two get removed?

Reply:
Because they were not mapped, and the comment asking how they should be mapped was left unanswered for 18 months. Using Google, I couldn't find any definitive answers for what these two values mean (as names, hexadecimal or decimal) in the context of a keyboard. Neither "World1" nor "World2" show up in the GLFW documentation, and neither do "GLFW_WORLD_1" or "GLFW_WORLD_2".

We can put them back in, but as long as they're not mapped, they're just hanging around for no reason.